src/main/java/com/indeed/operators/rabbitmq/resources/RabbitMQClusterFactory.java

```diff
@@ -169,7 +168,7 @@ private PodDisruptionBudget buildPodDisruptionBudget(final RabbitMQCustomResourc
                 )
                 .endMetadata()
                 .withNewSpec()
-                .withMaxUnavailable(new IntOrString(maximumDownPods))
+                .withMaxUnavailable(new IntOrString(1))
```

Comment:
What's the rationale here?  I know you ran into problems with the code as it was, but I can't remember the details.

Reply:
PodDisruptionBudget is immutable. If you if you changed the replica count from 3 to 4, maxUnvailable would go from 1 to 2 which would cause the API to throw an error.